code/__DEFINES/maths.dm

```diff
@@ -6,6 +6,7 @@
 
 #define PI						3.1416
 #define INFINITY				1e31	//closer then enough
+#define inf_win_lin					1.#INF //only for isinf test
```

Comment:
Please use something more descriptive like `system_type_infinity`. That example in particular would match BYOND's existing `system_type` for specifying the OS of the host machine.

Reply:
ok